src/styles/variables.scss

```diff
@@ -0,0 +1,8 @@
+$white: #fff;
+$main-accent: #060B35;
+$title-hover-color: #34568b;
+$secondary-accent: #616070;
+$blue-accent: #00ACDC;
```

Comment:
Add `-color` to all names -> `$title-hover-color`
Also don't use color name in variable name -> `white` is bad name